gatsby_fp/package.json

```diff
@@ -44,6 +44,7 @@
     "shortid": "^2.2.16"
   },
   "devDependencies": {
+    "@netlify/plugin-gatsby": "^1.0.3",
```

Comment:
So that I am understanding this properly, adding this plugin is sufficient to enable caching?

Reply:
Yes, just having this plugin installed cuts our netlify build times in half